content/en/monitors/faq/anomaly-monitor.md

```diff
@@ -0,0 +1,93 @@
+
+For example, the metric `app.requests` is noisy but has a constant average value of 8. On one day, there is a 10-minute anomalous period, starting a 9:00, during which the metric has an average value of 10. The graph below shows a series with a one-day time window; each point in the graph summarizes 5 minutes.
+
+{{< img src="monitors/monitor_types/anomaly/disappearing_day.png" alt="disappearing day" responsive="true" style="width:70%;" >}}
+
+The gray band here makes sense. It is wide enough to capture the noise in the timeseries, yet narrow enough that the anomaly at 9:00 stands out clearly. The next chart shows a zoomed-in view of a half-hour time window that includes the 10-minute anomaly; each point in the graph summarizes 10 seconds.
+
+{{< img src="monitors/monitor_types/anomaly/disappearing_half_hour.png" alt="disappearing half hour" responsive="true" style="width:70%;" >}}
+
+Again, the band seems to be reasonably sized, because the non-anomalous data from 8:50 - 9:00 and from 9:10 - 9:20 is inside the band. A band any narrower would start to highlight normal data as anomalous. Notice the band in this graph is ~8x wider than the one in the previous graph. The anomalous period from 9:00 - 9:10 looks different from the rest of the series, but it is not extreme enough to fall outside of the band.
+
+In general, if an anomaly disappears when you zoom in, this doesn't mean that it's not an anomaly. It means, while the individual points in the zoomed-in view are not anomalous in isolation, many slightly unusual points occurring together is anomalous.
+
+### Why do I get a query parsing error when trying to combine some functions with anomaly detection?
+
+Not all functions may be nested inside of calls to the `anomalies()` function. In particular, you may not include any of the following functions in an anomaly detection monitor or dashboard query: `cumsum()`, `integral()`, `outliers()`, `piecewise_constant()`, `robust_trend()`, or `trend_line()`.
+
+Anomaly detection uses historical data to establish a baseline of normal behavior for a series. The functions above are sensitive to the placement of the query window. The value of the series at a single timestamp can change significantly based upon where it falls within the query window. This sensitivity prevents anomaly detection from determining a consistent baseline for the series.
+
+### What happened to the adaptive algorithm?
+
+Previously, Datadog used an algorithm called `adaptive`, which would try to figure out a metric's inherent seasonality and adjust the predictions accordingly. Now, the seasonality is automatically detected when setting up a monitor. There is less need for this algorithm, which was slower and required more data than the other algorithms. Existing monitors that use the `adaptive` algorithm are untouched and continue to work.
```

Comment:
```suggestion
Datadog has evolved our algorithms to no longer need to use the adaptive algorithm. Existing monitors that use the `adaptive` algorithm are untouched and continue to work.
```
the key information here is that existing monitors still work. otherwise we should stay away from explaining our dev decisions

Reply:
Sounds good. Updating to change the wording slightly.